frontend/packages/dev-console/src/components/topology/const.ts

```diff
@@ -1,3 +1,5 @@
+import { STORAGE_PREFIX } from '@console/shared';
```

Comment:
Why is this prefix `@console/shared`?

Reply:
All the values that we are storing in localStorage have prefix ```bridge```. That is why I added it.